developers/academy/zero_to_mvp/102_queries_1/20_vector_parameters.mdx

```diff
@@ -178,23 +178,17 @@ The principle with all of these operators are the same, which is to retrieve out
 
 ## <i class="fa-solid fa-square-chevron-right"></i>&nbsp;&nbsp;Adding a threshold
 
-As vector search is based on similarity, it does not inherently exclude any of the results. Accordingly, some threshold is required to limit the number of results. Otherwise, a minimum of the number of available results and a Weaviate-defined maximum will be returned.
-
-Use of `limit` or distance thresholds are two ways to do restrict the number of results.
-
-### <i class="fa-solid fa-code"></i>&nbsp;&nbsp;`limit` threshold
-
-You have seen these above. The `limit` argument is used to limit the number of objects returned.
+Vector search is based on similarity, it does not inherently exclude any of the results. By default, vector search returns up to `QUERY_MAXIMUM_RESULTS`. To limit the number of results, set a threshold or limit.
```

Comment:
I think I would prefer to not tie this explanation to the particular env variable. 

Some users (e.g. WCS users) will not have `QUERY_MAXIMUM_RESULTS` exposed them to configure.

```suggestion
Vector search is based on similarity, it does not inherently exclude any of the results. By default, vector search returns up to a Weaviate-defined maximum number of results. To further limit the number of results, set a threshold or limit.
```

Reply:
Updated. I changed the wording slightly since some users can set QUERY_MAXIMUM_RESULTS